BUILD.bazel

```diff
@@ -36,7 +36,8 @@ updatesrc_update_all(
     name = "update_all",
     targets_to_run = [
         "@rules_bazel_integration_test//tools:update_deleted_packages",
-        "//doc:update",
+        # GH120: Enable once stardoc supports repo mapping.
+        # "//doc:update",
```

Comment:
You can do something like [this](https://github.com/aspect-build/bazel-lib/blob/main/lib/private/docs.bzl#L23) if you don't want to disable the target.

Reply:
Great idea. I updated `cgrindel_bazel_starlib` to support this. Once the release gets into BCR, I will update this PR with the new version.

Reply:
Done.